tests/smoke/smoke2.py

```diff
@@ -216,7 +216,9 @@ def debug_influx(node, influx_user, influx_pass):
         print("[WARNING]: InfluxDB Python Package is not installed!")
         return 1
     fail = check_port(node, 8086)
-    fail |= check_port(node, 8090)
+    # Port 8090 is specific to the java persister. Do not check
+    # port 8090 in order to allow the python persister to pass
+    # fail |= check_port(node, 8090)
```

Comment:
Yes, we can create a jinja2 template for smoke_configs.py which can be populated by Ansible or the devstack build process with the persister and API types (Java or Python). This has been done for smoke2_configs.py(.j2), which we will soon need to extend to include options for the Cassandra PoC. I’m happy to follow any recommendations.

> On Nov 3, 2015, at 9:10 PM, Joe Keen <notifications@github.com> wrote:
> 
> In tests/smoke/smoke2.py <https://github.com/hpcloud-mon/monasca-ci/pull/8#discussion_r43838809>:
> 
> > @@ -216,7 +216,9 @@ def debug_influx(node, influx_user, influx_pass):
> >          print("[WARNING]: InfluxDB Python Package is not installed!")
> >          return 1
> >      fail = check_port(node, 8086)
> > -    fail |= check_port(node, 8090)
> > +    # Port 8090 is specific to the java persister. Do not check
> > +    # port 8090 in order to allow the python persister to pass
> > +    # fail |= check_port(node, 8090)
> Is there a way to see if we're running the Python persister and leave this check in for the Java persister?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/hpcloud-mon/monasca-ci/pull/8/files#r43838809>.
>